auto_yaw: false
read_wrench: 0
compensate_d: true
ref_traj: /root/catkin_ws/src/bluerov2/bluerov2_mpc/traj/circle.txt
applied_forcex: /root/catkin_ws/src/bluerov2/bluerov2_dobmpc/config/forcex.txt
applied_forcey: /root/catkin_ws/src/bluerov2/bluerov2_dobmpc/config/forcey.txt
applied_forcez: /root/catkin_ws/src/bluerov2/bluerov2_dobmpc/config/forcez.txt
applied_torquez: /root/catkin_ws/src/bluerov2/bluerov2_dobmpc/config/torquez.txt
yaml_name: /config/gazebo_param.yaml